css/style.css

```diff
@@ -1,47 +1,54 @@
-/* body {
-	max-width: 940px;
-	margin: 0 auto;
-} */
+body {
+	font-family: Georgia, 'Times New Roman', Times, serif;
+}
 a {
 	text-decoration: none;
 }
+header {
+	display: flex;
+	justify-content: center;
+}
 header div{
 	display: flex;
+	justify-content: space-between;
+	align-items: center;
 	width: 913.5px;
 	height: 137.3px;
 }
 #logo{
-	display: block;
-	width: 287.3px;
+	width: 287px;
 }
 nav {
 	display: flex;
-	font-size: 20px;
+	justify-content: space-around;
+	font-size: 14px;
+	width: 520px;
 	height: 39px;
-}
-nav span {
-	display: flex;
-	line-break: auto;
-	font-size: 10px;
-	font-weight: normal;
+	font-family: Georgia, 'Times New Roman', Times, serif;
 }
 nav a {
 	font-weight: bold;
 	border-left:1px dotted #d5d5d5;
-	padding-left: 10.5px;
+	padding-left: 10px;
 }
-nav a:link {
-	color: #666666;
+nav span {
+	display: flex;
+	line-break: auto;
```

Comment:
This is a default value, and there is no need to give default value. This line is redundant.